lib/crypto.js

```diff
@@ -0,0 +1,86 @@
+import {
+  sign,
+  verify,
+  createPrivateKey,
+  createPublicKey
+} from 'crypto';
+import {asn1, ed25519, util} from 'node-forge';
+import {privateKeyDerEncode, publicKeyDerEncode} from './util.js';
+
+const {privateKeyFromAsn1, publicKeyFromAsn1} = ed25519;
+const {ByteBuffer} = util;
+const publicKeyEncoding = {format: 'der', type: 'spki'};
+const privateKeyEncoding = {format: 'der', type: 'pkcs8'};
+
+// FIXME someone more familar with TypedArrays and Buffers
+// should review this
+const typedArrayToBuffer = array => {
```

Comment:
@davidlehn does this look correct to you?

Reply:
Addressed this here: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/3/commits/7efbbfd8d01fe357a5593ec4095533ea8d108c96